dagger1support/src/test/java/mortar/MortarActivityScopeTest.java

```diff
@@ -636,4 +636,33 @@ void create(Bundle bundle) {
 
     getBundleServiceRunner(activityScope).onSaveInstanceState(new Bundle());
   }
+
+  // Make sure that when a scope dies, a new scope with the same name doesn't
+  // accidentally receive the old one's bundle.
+  @Test public void endScopeEndBundle() {
+    MyBundler fooBundler = new MyBundler("fooBundler") {
+      @Override public void onLoad(Bundle savedInstanceState) {
+        assertThat(savedInstanceState).isNull();
```

Comment:
This assert is out of place. It's unclear when it will be triggered in the test.

How about mocking the bundler instead, and using verify() in the right place?